### PR TITLE
Entities via app, extended with timespan type

### PR DESCRIPTION
**Pull request by lbiedinger**
Using the new app, requires a migration for the timespans to be accepted on Automated entity cards.
This migration also adds the app to the relevant contentTypes.

### CODE REVIEW

File: src/pages/contentful/entity-harvester/index.vue [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        case 'Concept':
+          // use `note`
+          // NB: language map with each value being an array of literals
+          return langMapValueForLocale(response.note, 'en').values[0];
+        case 'Organization':
+          // use `description`
+          // NB: language map with each value being a single literal
+          return langMapValueForLocale(response.description, 'en').values[0];
+        case 'Timespan':
+          // TODO: use what? `${response.begin} to ${response.end}`?
```
Comment (mirjamverloop) [inline src/pages/contentful/entity-harvester/index.vue:163]: Begin to end sounds a bit "obvious", let's keep it blank as you have right now, so an editor can add their own description? Or is there a more descriptive description available in the API?

File: src/pages/contentful/entity-harvester/index.vue [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+          this.showError(entityResponse.error);
+        }
+      },
+
+      entityParamsFromUrl(url) {
+        url = url.replace(new RegExp('^https?://api.europeana.eu/entity'), BASE_URL);
+        if (isEntityUri(url)) {
+          return entityParamsFromUri(url);
+        }
+        const pageMatch = url.match(new RegExp('^https?://[^/]+(/[a-z]{2})?/collections/(person|topic|time)/([0-9]+)+'));
```
[the remark this answers is not in the document]
Comment (lbiedinger) [reply src/pages/contentful/entity-harvester/index.vue:94]: the isEntityUri function only accounts for concept/agent/place/timespan at the moment, so those are the only "pages" in the portal.

File: src/migrations/0043-entity-harvest-app.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  // "Entity harvester - Europeana" app (pre-installed in space & env)
+  automatedEntityCard.removeSidebarWidget('extension', 'europeanaEntityHarvester');
+  automatedEntityCard.addSidebarWidget('app',  process.env.ENTITY_HARVEST_APP_ID, {}, 'content-preview-widget');
+  automatedEntityCard
+    .editField('identifier')
+    .validations([
+      {
+        regexp: {
+          pattern:
+            '^http://data\\.europeana\\.eu/(agent|concept|organization|place|timespan)(/base)?/[0-9]+$',
```
[the remark this answers is not in the document]
Comment (lbiedinger) [reply src/migrations/0043-entity-harvest-app.js:18]: Yes, the API uses 'organization'.

File: src/pages/contentful/entity-harvester/index.vue (hunk 1 above)
Comment (lbiedinger) [reply src/pages/contentful/entity-harvester/index.vue:163]: nothing I'm aware of/have seen on the examples I've checked.

**Review by mirjamverloop — APPROVED** [review]

**Comment by lbiedinger** [comment]
redundant due to #1123

**Pull request closed without merging**